### ISSUE TITLE
[newpct] Unexpected character encountered while parsing value: <. Path

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#6551]
Jackett Version 0.12.1115.0
Tracker: newpct
Search: 1234abcd

```
Manual search for "1234abcd" on newpct with 0 results.
Jackett.Common.IndexerException: Exception (newpct): Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
 ---> System.Exception: Parse error
 ---> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at Jackett.Common.Indexers.Newpct.ParseSearchJsonContent(Uri uri, String content) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Newpct.cs:line 741
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 846
   at Jackett.Common.Indexers.Newpct.ParseSearchJsonContent(Uri uri, String content) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Newpct.cs:line 789
   at Jackett.Common.Indexers.Newpct.MovieSearch(Uri siteLink, TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Newpct.cs:line 639
   at Jackett.Common.Indexers.Newpct.PerformQuery(Uri siteLink, TorznabQuery query, Int32 attempts) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Newpct.cs:line 333
   at Jackett.Common.Indexers.Newpct.PerformQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Newpct.cs:line 175
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 353
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 353
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 823
   at Jackett.Server.Controllers.ResultsController.Results(ApiSearch requestt) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 227
```

**Comment by garfield69** [comment]
@snamds  as the last C# volunteer to maintain this indexer, would you like to have a go fixing this?
apparently the indexer C# code does not seem to have a working handle when there are no results from a search request... Thanks.

**Comment by garfield69** [comment]
fix included in jackett 0.12.1132
